apisix/plugins/log-rotate.lua

```diff
@@ -210,7 +210,7 @@ local function rotate_file(files, now_time, max_kept, timeout)
         return
     end
 
-    local new_files = core.table.new(2, 0)
+    local new_files = core.table.new(#files, 0)
```

Comment:
seems like unrelated change.

Reply:
`new_files` stores the name of access.log and error.log. If `enable_access_log` is set to false, nginx will not generate `access.log` file, no need to pre-allocate an array of length 2.